pkg/ipamd/datastore/data_store_test.go

```diff
@@ -96,6 +96,57 @@ func TestDeleteENI(t *testing.T) {
 	assert.Error(t, err)
 	err = ds.RemoveENIFromDataStore("eni-1", true)
 	assert.NoError(t, err)
+
+}
+
+func TestDeleteENIwithPDEnabled(t *testing.T) {
```

Comment:
nit: this is nearly the same test as above. Can you share the test above and pass a `PDEnabled` boolean?